components/Storyblok/SbContentMenu/SbContentMenuParentItem.tsx

```diff
@@ -36,7 +36,7 @@ export const SbContentMenuParentItem = ({ blok, slug }: SbContentMenuParentItemP
       >
         {parentItemText}
       </CtaLink>
-      <CreateBloks blokSection={nestedMenu} />
+      <CreateBloks blokSection={nestedMenu} slug={slug} />
```

Comment:
Forgot to pass the slug to nested content menus previously. See 
https://deploy-preview-511--adapt-giving.netlify.app/planned-giving/pooled-income-funds
The 2nd level menu active link item now has the black text color and bar